docs/content/preview/reference/configuration/_index.md

```diff
@@ -57,7 +57,7 @@ type: indexpage
         <div class="title">Default ports</div>
       </div>
       <div class="body">
-        Default ports for APIs, RPC and admin web servers.
+        Default ports for APIs, RPC, and admin web servers.
```

Comment:
Technically Remote Procedure Call is singular, so RPCs would be correct here (unless it's an _RPC web server_ and an _admin web server_...)
```suggestion
        Default ports for APIs, RPCs, and admin web servers.
```